tgui/packages/tgui/interfaces/GreyscaleModifyMenu.tsx

```diff
@@ -35,6 +35,17 @@ enum Directions {
   NorthWest = "northwest"
 }
 
+const DirectionAbbreviation : Map<Direction, string> = new Map([
+  [Direction.North, "N"],
+  [Direction.NorthEast, "NE"],
+  [Direction.East, "E"],
+  [Direction.SouthEast, "SE"],
+  [Direction.South, "S"],
+  [Direction.SouthWest, "SW"],
+  [Direction.West, "W"],
+  [Direction.NorthWest, "NW"],
+]);
+
```

Comment:
You can use a simple object for this which has less overhead:

```ts
const DirectionAbbreviation: Record<Direction, string> = {
  [Direction.North]: "N",
  [Direction.NorthEast]: "NE",
  ...
};
```

and access it like this

```
DirectionAbbreviation[Direction.North]
```